Common/CommonAssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyConfiguration("")]
 
 // If you change this version, make sure to change Build\build.proj accordingly
-[assembly: AssemblyVersion("97.0.0.0")]
-[assembly: AssemblyFileVersion("97.0.0.0")]
+[assembly: AssemblyVersion("98.0.0.0")]
```

Comment:
Do you need to revise this version before sending PR to main branch ?

Reply:
I'm not sure how this gets changed. For now, I will revert this.